apps/deploy-web/src/store/networkStore.ts

```diff
@@ -66,8 +69,42 @@ export const initiateNetworkData = async () => {
   );
 };
 
-const selectedNetwork = atom<Network>(networks[0]);
+const selectedNetworkId = atomWithStorage<Network["id"]>(
+  "selectedNetworkId",
+  defaultNetworkId,
```

Comment:
So this sets the default value in the localStorage on first execution if not set?

Reply:
it is not, but it wouldn't matter if we didn't have direct local storage accesses. Just realise that. Will add some logic so it does set it till that task is done.

Reply:
ok, here it is